fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/VehicleUnit.java

```diff
@@ -99,6 +103,7 @@ public VehicleUnit(String vehicleName, VehicleType vehicleType, final GeoPoint i
         }
 
         basicSensorModule = new EnvironmentBasicSensorModule();
+        defaultLidarSensorModule = new DefaultLidarSensorModule(this.getId());
```

Comment:
How would we switch to a non-default `LidarSensorModule`?

Reply:
It's currently not intended to switch lidar modules, we just need an implementation of the API interface.